.github/workflows/cypress-workflow.yml

```diff
@@ -7,9 +7,9 @@ on:
     branches:
       - "*"
 env:
-  OPENSEARCH_DASHBOARDS_VERSION: '2.x'
-  OPENSEARCH_VERSION: '2.7.0-SNAPSHOT'
-  SECURITY_ANALYTICS_BRANCH: '2.x'
+  OPENSEARCH_DASHBOARDS_VERSION: 'main'
+  OPENSEARCH_VERSION: '3.0.0-SNAPSHOT'
```

Comment:
Recommend to remove the usage of this variable

Reply:
Ah, thanks for the tip! I'll add this as a follow-up action.